web/resources/lib/components/UserInfo.svelte

```diff
@@ -0,0 +1,77 @@
+<script lang="ts">
+  import WikijumpAPI, { identity as currentIdentity } from "@wikijump/api"
+  import { Sprite } from "@wikijump/components"
+  import type { UserIdentity } from "@wikijump/api"
+
+  export let user: null | string | number = null
+
+  export let noavatar = false
+
```

Comment:
Personally I prefer positive variables, like `hasAvatar`, which avoids double-negative constructions like `!noAvatar`.

Reply:
Fully agree, this is just a thing with Svelte. If this was instead `avatar` and it defaulted to `true`, you'd to do this:
```svelte
<UserInfo avatar={false} />
```
versus:
```svelte
<UserInfo noavatar />
```

When you chain multiple of these boolean attributes together it makes things more readable. I think there is a proposal for adding `!avatar` as a valid attribute, but I have no idea if that got anywhere.